Tests/SwiftStdlibTests/ArrayExtensionsTests.swift

```diff
@@ -8,14 +8,25 @@
 import XCTest
 @testable import SwifterSwift
 
-private struct Person: Equatable {
+private struct Person: Equatable, Hashable {
     var name: String
     var age: Int?
+    var location: Location
 
-    static func == (lhs: Person, rhs: Person) -> Bool {
-        return lhs.name == rhs.name && lhs.age == rhs.age
+    init(name: String, age: Int?, location: Location = Location(city: "New York")) {
```

Comment:
```suggestion
    init(name: String, age: Int?, location: Location = nil) {
```
`location` should be optional. Is that possible?

Reply:
Yes.